### PR TITLE
acf: add second backup production key

### PR DESCRIPTION
**Pull request by joseph-reynolds**
This enhances the ACF service to use the second backup production key when checking new ACFs.

Tested:
Uploading ACFs still works.
Did not test with an ACF created with the second backup production key.

Signed-off-by: Joseph Reynolds <joseph-reynolds@charter.net>

### CODE REVIEW

File: bmc-acf/acf_manager.cpp [hunk 1]
```diff
@@ -394,7 +397,21 @@ acf_info ACFCertMgr::installACF(std::vector<uint8_t> accessControlFile)
         }
         else
         {
-            log<level::ERR>("cannot read production key file");
+            log<level::ERR>("cannot read production backup key file");
+            elog<InternalFailure>();
+        }
+    }
+    if (prodBackup2KeyExists && sRc != CeLogin::CeLoginRc::Success)
```
Comment (geissonator) [inline bmc-acf/acf_manager.cpp:404]: nit: best to use () to explicitly show logic precedence for review ease and to avoid dangerous bugs.

File: bmc-acf/acf_manager.cpp [hunk 2]
```diff
@@ -394,7 +397,21 @@ acf_info ACFCertMgr::installACF(std::vector<uint8_t> accessControlFile)
         }
         else
         {
-            log<level::ERR>("cannot read production key file");
+            log<level::ERR>("cannot read production backup key file");
+            elog<InternalFailure>();
```
Comment (geissonator) [inline bmc-acf/acf_manager.cpp:401]: nit: InternalFailure is something we try to avoid in the community. Also, this path doesn't seem related to the new prodBackup2KeyExists variable? Wont an InternalFailure also throw and unhandled exception which means we don't try the new backup below?

File: bmc-acf/acf_manager.cpp (hunk 1 above)
Comment (joseph-reynolds) [reply bmc-acf/acf_manager.cpp]: Added parens

File: bmc-acf/acf_manager.cpp (hunk 2 above)
Comment (joseph-reynolds) [reply bmc-acf/acf_manager.cpp:401]: This really is an internal error.  The code above already check if the public key file exists, so it should be readable.  I've never seen this happen in practice.  And I don't want to change the control flow.  So I didn't change it.

**Review by geissonator — APPROVED** [review]

**Comment by joseph-reynolds** [comment]
SW551437

**Comment by geissonator** [comment]
PR has conflicts and is failing CI. Looks like CI failures are formatting related. Some fails may not be related to this change but just new checks in the CI scripts.

**Comment by joseph-reynolds** [comment]
The CI failure is apprently:

**Comment by joseph-reynolds** [comment]
Bumped by https://github.ibm.com/openbmc/openbmc/pull/2580
